easybuild/easyblocks/s/scipy.py

```diff
@@ -165,10 +174,17 @@ def test_step(self):
             change_dir(tmp_builddir)
 
             # reconfigure (to update prefix), and install to tmpdir
-            MesonNinja.configure_step(self, cmd_prefix=tmp_installdir)
```

Comment:
We should look into why the `cmd_prefix` didn't work as intended, but that can be done in a follow-up PR...

`cmd_prefix` isn't used at all in `MesonNinja.configure_step` currently, so we need to check what's going on there.